Review user guide (#979)
ansible/receptor: docs/source/contributing.rst

An initial checklist for your change to increase the likelihood of acceptance:
- No issues when running linters/code checkers
- No issues from unit/functional tests

> - Write good commit messages. See How to write a Git commit message<https://cbea.ms/git-commit/>.

Reviewer: ```suggestion
- Write good commit messages. See `How to write a Git commit message<https://cbea.ms/git-commit/>`_.
```

Reviewer 2: I would also qualify what we mean by good commit messages so it's not so subjective. Perhaps something like: "Write descriptive and meaningful commit messages."

I also like this resource (which also refers to "good commit messages" lol): https://gist.github.com/webknjaz/cb7d7bf62c3dda4b1342d639d0e78d79

Author: Thank you for the clarifications.  I've added them in the next set of commits

> - Write desriptive and meaningful commit messages. See `How to write a Git commit message <https://cbea.ms/git-commit/>`_ and `Learn to write good commit message and description <https://gist.github.com/webknjaz/cb7d7bf62c3dda4b1342d639d0e78d79>`_.

Reviewer 2: ```suggestion
- Write desriptive and meaningful commit messages.
  See `How to write a Git commit message <https://cbea.ms/git-commit/>`_ and `Learn to write good commit message and description <https://gist.github.com/webknjaz/cb7d7bf62c3dda4b1342d639d0e78d79>`_.
```

> - Write desriptive and meaningful commit messages. See `How to write a Git commit message <https://cbea.ms/git-commit/>`_ and `Learn to write good commit message and description <https://gist.github.com/webknjaz/cb7d7bf62c3dda4b1342d639d0e78d79>`_.

Reviewer 3: ```suggestion
- Write descriptive and meaningful commit messages. See `How to write a Git commit message <https://cbea.ms/git-commit/>`_ and `Learn to write good commit message and description <https://gist.github.com/webknjaz/cb7d7bf62c3dda4b1342d639d0e78d79>`_.
```

After the release is published, the `Promote Release <https://github.com/ansible/receptor/actions/workflows/promote.yml>`_ workflow will run automatically. This workflow will:

- Publish ``receptorctl`` to `PyPI <https://pypi.org/project/receptorctl/>`_.
- Pull the container image from ghcr.io, re-tag, and push to `Quay.io <https://quay.io/repository/ansible/receptor>`_.
- Build binaries for various OSes/platforms, and attach them to the `release <https://github.com/ansible/receptor/releases>`_.

.. note::

>   If `Stage Release`_ needs to be re-run more than once, delete the tag beforehand to prevent the workflow from failing.

Reviewer 3: I'd leave the first part of this sentence alone, just to avoid the passive voice:

```suggestion
  If you need to re-run `Stage Release`_ more than once, delete the tag beforehand to prevent the workflow from failing.
```
